.github/workflows/publish-noir-js.yml

```
@@ -17,11 +17,15 @@ jobs:
       - name: Install jq
         run: sudo apt-get install jq
 
-      - name: Install cargo and wasm-bindgen
-        run: |
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-          source $HOME/.cargo/env
-          cargo install -f wasm-bindgen-cli --version 0.2.86
+      - name: Setup toolchain
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: 1.67.0
```

```suggestion
        uses: dtolnay/rust-toolchain@1.66.0
```

We don't want to increase the MSRV.